File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/library/repository/permissions/storage/request/IStoragePermissionsRequestNotificationBuilder.java
package com.lasthopesoftware.bluewater.client.library.repository.permissions.storage.request;

import android.app.Notification;

/**
 * Created by david on 7/10/16.
 */
public interface IStoragePermissionsRequestNotificationBuilder {
	Notification buildStoragePermissionsRequestNotification(int libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/library/request/IStoragePermissionsRequestNotificationBuilder.kt
package com.lasthopesoftware.bluewater.client.browsing.library.request

import android.app.Notification

/**
 * Created by david on 7/10/16.
 */
interface IStoragePermissionsRequestNotificationBuilder {
    fun buildStoragePermissionsRequestNotification(libraryId: Int): Notification?
}
